.github/workflows/main.yml

```diff
@@ -22,12 +22,12 @@ jobs:
     strategy:
       matrix:
         os: [Ubuntu, MacOS, Windows]
-        python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9, "3.10"]
```

Comment:
Just for consistency, I would recommend we quote all values now.

Reply:
done